certified-connectors/Yeeflow/apiDefinition.swagger.json

```diff
@@ -990,6 +972,269 @@
         ],
         "description": "Delete a trigger"
       }
+    },
+    "/lists/41/{listID}/items/{id}/files": {
+      "post": {
+        "consumes": ["application/json"],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "Data": {
+                  "type": "string",
+                  "description": "ID of modified item",
+                  "x-ms-summary": "Item ID"
+                },
+                "Status": {
+                  "type": "integer",
+                  "description": "0 is success",
+                  "format": "int32"
+                },
+                "Message": {
+                  "type": "string",
+                  "description": "Error message"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Add file to item",
+        "operationId": "AddItemFile",
+        "parameters": [
+          {
+            "$ref": "#/parameters/application"
+          },
+          {
+            "$ref": "#/parameters/listID"
+          },
+          {
+            "$ref": "#/parameters/itemID"
+          },
+          {
+            "in": "query",
+            "name": "FieldID",
+            "type": "string",
+            "x-ms-summary": "Field",
+            "description": "Select a field",
+            "x-ms-dynamic-values": {
+              "operationId": "GetListFields",
+              "value-path": "FieldName",
+              "value-title": "DisplayName",
+              "value-collection": "Data",
+              "parameters": {
+                "application": "",
```

Comment:
@lulut I might be missed this in first review, Please let us know why application value is given as empty string? If it is not intentional, you can remove this property.

Rest everything looks good.

Reply:
"application" is a parameter to execute GetListFields operation.  The API has default value for this parameter if its empty.  It is left here to assign values when later needed.  Can we keep this?